### DISCUSSION TITLE
Making junction collection system generated fields editable

### DISCUSSION THREAD

**Discussion by adanielyan** [directus/directus#15545] [Archive]
This is a follow up discussion of #15462 

I have a M2A relationship in one of my collections. In the M2A field Relationship settings I selected Delete ***/*** item (cascade) option under On Deselect of field_name. However, when I deselect the related item the record is deleted only from the Junction Collection, but not from the related collection. So the related item itself is not deleted.

I believe this can be resolved by configuring the `item` field in a junction collection to delete the related item when the record is deleted from junction collection. However the item field in junction collection cannot be edited and saved. Is there a way to unlock the system generated fields?

https://user-images.githubusercontent.com/861777/189789950-dfa0d7f0-54bd-4861-ab99-b024aa23d1eb.mov

**Comment by rijkvanzanten** [comment]
There's currently no relational triggers for many to any relationships, as those relationships don't "exist" in the database, and thus don't have the on update/on delete triggers
